init

```diff
@@ -20,6 +22,23 @@ if [ "$NODE_ROLE" = "it-hybrid" ]; then
   sudo ${SCRIPTDIR}/itv1/util/etcd2-setup.sh $SCRIPTDIR
 fi
 
+if [ "$NODE_ROLE" = "control" ];then
```

Comment:
This is definitely going to have to change. @matthewdfuller has a post-etcd-setup phase I think in ethos-systemd where this logic can be run as a separate script...?